cmd/minishift/cmd/start.go

```diff
@@ -127,6 +128,12 @@ func runStart(cmd *cobra.Command, args []string) {
 	setOcProxy()
 	setShellProxy()
 
+	if viper.IsSet("http-proxy") {
```

Comment:
I have not found any documentation for subscription manager that it uses https. I suggest we create an followup issue and findout more about it.